models/engine/file_storage.py

```diff
@@ -88,3 +88,22 @@ def close(self):
             calls the reload() method for deserialization from JSON to objects
         """
         self.reload()
+
+    def get(self, cls, id):
+        """
+            retrieves one object based on class name and id
+        """
+        if cls and id:
+            for obj in self.all(cls).values():
+                if obj.id == id:
+                    return obj
+        return None
+
+    def count(self, cls=None):
+        """
+        count of all objects in storage
+        """
+        if cls is None:
+            return (len(self.all(cls).values()))
+        else:
+            return (len(self.all().values()))
```

Comment:
For empty dictionary, this returns: `len({}.values())`.  Could this be a potential problem?